cilium/cmd/cleanup.go

```diff
@@ -0,0 +1,242 @@
+// Copyright 2017 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/cilium/cilium/common"
+	"github.com/cilium/cilium/daemon/defaults"
+	"github.com/cilium/cilium/pkg/bpf"
+
+	"github.com/spf13/cobra"
+	"github.com/vishvananda/netlink"
+)
+
+// configCmd represents the config command
+var cleanupCmd = &cobra.Command{
+	Use:   "cleanup",
+	Short: "Reset the agent state",
+	Run: func(cmd *cobra.Command, args []string) {
+		common.RequireRootPrivilege("cleanup")
+		runCleanup()
+	},
+}
+
+var force bool
+
+const (
+	ciliumLinkPrefix = "cilium_"
```

Comment:
This shouldn't block this PR but are these prefixes/names defined already in the relevant packages, and would it be more robust to export them from there and use them here?

Reply:
Will add to my todo list for replacing init.sh

Reply:
Yes, good idea to clean this up as we go go-native with these interactions.